SPEC_MAINTAINER_PROTOCOL.md

```diff
@@ -0,0 +1,80 @@
+
+- Writting the spec and iterating on it to the point that the spec and the code progress smoothly together.
+- For mature and more stable specs, have tests (using [Protocol Driven Development](https://github.com/ipfs/pdd), [TLA+](https://lamport.azurewebsites.net/tla/tla.html) or another framework of verification) to ensure that all implementations stand correct.
+
+The very aspirational future is to have a spec language in which we can describe how the program should function and it will automatically create the tests to verify it. The aspiration future ++ is having such spec language that writes the code itself and verifies it 🚀.
+
+The Spec Maintainer Protocol takes a large amount of inspiration from the repo [Lead Maintainer Protocol](https://github.com/ipfs/team-mgmt/blob/master/LEAD_MAINTAINER_PROTOCOL.md).
+
+## Maintaining a Specification
+
+A Specification Maintainer is a contributor that has shown that understand the subsystem of the spec that is maintaining and that is passionate about documenting it well. There is a benefit for this contributor to be actively involved with the development of the subsystem at least in one or more of the languages.
+
+Each spec should be maintained at least by 1 person and up to 3 for efficiency purposes.
+
+The spec maintainer(s) are recognized at the top of the specification document on the list "Maintainers(s)" and these are the people that should be requested to review a PR to that specific spec.
+
+### Responsibilities
+
+As a Spec Maintainer, you are expected to:
+
+- Respect and follow the [IPFS Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+- Have a great understanding of the subsystem  purpose and how it is used by other parts of the project.
+- Review and merge PRs to the Spec
+- Respond in a timely manner to Github issues on ipfs/specs repo that are related to the spec being maintained.
+- Notify the developers of the subsystem in case there is a new proposal for a change or also request the developers to notify when there is a system change that needs to be reflected on the spec.
+
+### Specification Status
+
+The Specification Status description can be found on the main [README of this repo](https://github.com/ipfs/specs#badges-and-spec-lifecycle).
+
+## Disclaimer as 2019 Q3
+
+As of 2019 Q3, the specs are vastly out of date and should not be relied uppon. We will run an effort to get these up to date and check the boxes here when it is done:
+
+- **IPFS Protocol:**
+  - [ ] [Protocol Architecture Overview](./ARCHITECTURE.md) - the top-level spec and the stack
+  - [ ] [Other IPFS Overviews](/overviews) - quick overviews of the various parts of IPFS
+- **User Interface (aka Public APIs):**
+  - [ ] [Core API (aka using IPFS as a package/module)](./API_CORE.md)
+    - [x] [JavaScript implementation details](https://github.com/ipfs/interface-js-ipfs-core)
+    - [ ] [Golang implementation details](https://github.com/ipfs/interface-go-ipfs-core)
```

Comment:
This is up-to-date and tested by go-ipfs's CI.

Reply:
Note these should be updated to whatever we decide here https://github.com/ipfs/specs/pull/214/files#r303472977